src/ResourceManager/Insights/Commands.Insights/Alerts/AddAzureRmAlertRuleCommandBase.cs

```diff
@@ -53,7 +53,7 @@ public abstract class AddAzureRmAlertRuleCommandBase : ManagementCmdletBase
         /// </summary>
         [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The resource group name")]
         [ValidateNotNullOrEmpty]
-        public string ResourceGroup { get; set; }
+        public string ResourceGroupName { get; set; }
```

Comment:
@gucalder please add an alias to `ResourceGroup` to avoid undocumented breaking changes

Reply:
I reverted the argument names to ResourceGroup and added an alias ResourceGroupName and removed the exceptions from BreakingChangesIssues.csv

Reply:
Alias added... everywhere including documentation